src/ClientApp/App.xaml.cs

```diff
@@ -86,7 +90,7 @@ private void App_RequestedThemeChanged(object sender, AppThemeChangedEventArgs e
 
     private void SetStatusBar()
     {
-        var nav = Current.MainPage as NavigationPage;
+        var nav = Windows[0].Page as NavigationPage;
```

Comment:
@jfversluis  thoughts?

Reply:
Yep that is unfortunately what it is for now. Alternatively since you're using Shell you could do `Shell.Current.CurrentPage` which is a tiny bit nicer? But also I think that might be going away at some point since it's also not great.

Additionally, I don't think the whole StatusBar thing does much anyway? 😅

https://github.com/dotnet/eShop/blob/main/src/ClientApp/Services/Theme/Theme.shared.cs#L5